FabNESO/tasks.py

```diff
@@ -105,11 +113,7 @@ def neso_ensemble(
             temporary_config_path = Path(temporary_config_directory)
             # Because FabSIM is a bit weird with commas, build the dict here
             parameter_scan_dict = {
-                parameter: (
-                    float(values.split("/")[0]),
-                    float(values.split("/")[1]),
-                    int(values.split("/")[2]),
-                )
+                parameter: (_parse_parameter_scan_string(values, ":"))
```

Comment:
```suggestion
                parameter: _parse_parameter_scan_string(values, ":")
```
Don't think the parenthesis here is doing anything